<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainScreen">
     <widget class="QPushButton" name="battleWindowButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>103</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Battle Window</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>478</x>
        <y>50</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>View Pokemon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createTeamButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create team</string>
      </property>
     </widget>
     <widget class="miwidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pokemonImage">
      <property name="geometry">
       <rect>
        <x>438</x>
        <y>20</y>
        <width>291</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="AttackAnimation" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="AttackAnimationE" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>40</y>
        <width>591</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>551</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="teamsDrop">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <zorder>pokemonImage</zorder>
     <zorder>widget</zorder>
     <zorder>widget_3</zorder>
     <zorder>widget_2</zorder>
     <zorder>battleWindowButton</zorder>
     <zorder>label</zorder>
     <zorder>pushButton</zorder>
     <zorder>createTeamButton</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>comboBox</zorder>
     <zorder>teamsDrop</zorder>
    </widget>
    <widget class="QWidget" name="CreateTeamScreen">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pokemon 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Hit me</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pokemon1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>hello</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionnew"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>miwidget</class>
   <extends>QWidget</extends>
   <header location="global">miwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttackAnimation</class>
   <extends>QWidget</extends>
   <header>attackanimation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttackAnimationE</class>
   <extends>QWidget</extends>
   <header>attackanimatione.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
